tests/test_cors.py

```diff
@@ -0,0 +1,457 @@
+import re
+import uuid
+
+import mock
+from nose.tools import raises
+
+import falcon
+from falcon.cors import CORS
+import falcon.testing as testing
+from falcon.testing.resource import TestResource
+
```

Comment:
The imports are grouped according to PEP8:  https://www.python.org/dev/peps/pep-0008/#imports